docs/integrators/observing-squad.md

```diff
@@ -72,6 +78,21 @@ In order to check the status of the Observing Squad, please see the section **Mo
 
 The Observing Squad can be updated using the Mainnet installation scripts.
 
+### **April 2021 upgrade**
+
+In order to upgrade the Observing Squad - that is, both the Observers and the Proxy, one should issue the following commands:
+
+```
+$ cd ~/elrond-go-scripts-mainnet
+$ ./script.sh github_pull
+$ ./script.sh stop
+$ ./script.sh upgrade_squad
+$ ./script.sh upgrade_proxy
+$ ./script.sh start
+```
+:::important
+The Observing Squad must be upgraded before 28th of April 14:30 UTC. Otherwise, the nodes will lose sync.
```

Comment:
The Observing Squad must be upgraded before 28th of April 14:30 UTC. Otherwise, the nodes will lose sync. - let's nod add time sensitive call to actions in documentation, that's for comms channels. Let's leave this bit out.